runtime/doc/editing.txt

```diff
@@ -1245,8 +1245,8 @@ If you want to always use ":confirm", set the 'confirm' option.
 			|:diffsplit|, |:diffpatch|, |:pedit|, |:redir|,
 			|:source|, |:update|, |:visual|, |:vsplit|,
 			and |:qall| if 'confirm' is set.
-			{only in Win32 GUI, in console `browse edit` works
-			if the FileExplorer autocommand group exists}
+			Note: only in Win32 GUI; in console `:browse edit`
+			works if the FileExplorer autocommand group exists.
```

Comment:
The `{only in blah}` stuff is no longer supported because that parses as an invalid `argument`. There were only a few remaining cases, anyway.